neo/Prompt/Commands/Send.py

```diff
@@ -31,6 +31,10 @@ def construct_and_send(prompter, wallet, arguments, prompt_password=True):
         address_to = get_arg(arguments, 1)
         amount = get_arg(arguments, 2)
 
+        if int(float(amount)) == 0:
```

Comment:
just `float(amount)` is enough, no need to `int()` as well
```
>>> float("0.0") == 0
True
>>> float("0") == 0
True
```

Reply:
Okay, thank you for the feedback! I really appreciate it!